### PR TITLE
Add some tests and collate for test

### PR DESCRIPTION
**Pull request by 592363789**

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/BookShelfParser.java [hunk 1]
```diff
@@ -149,10 +149,10 @@ private Command getCommand(String commandWord, String arguments) throws ParseExc
 
         case ListCommand.COMMAND_WORD:
             return new ListCommandParser().parse(arguments);
-
+        //@@author 592363789
```
Comment (takuyakanbr) [inline src/main/java/seedu/address/logic/parser/BookShelfParser.java]: Remove your changes to this file. Don't collate this kind of small things.

File: src/test/java/systemtests/EditCommandSystemTest.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 package systemtests;
-
+//@@author 592363789
```
Comment (takuyakanbr) [inline src/test/java/systemtests/EditCommandSystemTest.java]: Remove your changes to this file. You didn't write this file.

File: src/main/java/seedu/address/logic/parser/BookShelfParser.java (hunk 1 above)
Comment (592363789) [reply src/main/java/seedu/address/logic/parser/BookShelfParser.java]: How to remove your changes to the generated `.md` files of others?

File: src/main/java/seedu/address/logic/parser/BookShelfParser.java [hunk 3]
```diff
@@ -164,19 +164,19 @@ private Command getCommand(String commandWord, String arguments) throws ParseExc
 
         case SelectCommand.COMMAND_WORD:
             return new SelectCommandParser().parse(arguments);
-
+        //@@author 592363789
```
Comment (takuyakanbr) [inline src/main/java/seedu/address/logic/parser/BookShelfParser.java]: Remove the changes in this file...?

File: src/main/java/seedu/address/logic/parser/BookShelfParser.java [hunk 4]
```diff
@@ -164,19 +164,19 @@ private Command getCommand(String commandWord, String arguments) throws ParseExc
...
         case SetPasswordCommand.COMMAND_WORD:
             return new SetPasswordCommandParser().parse(arguments);
-
+        //@@author
```
Comment (takuyakanbr) [inline src/main/java/seedu/address/logic/parser/BookShelfParser.java]: Remove the changes in this file...?

File: src/main/java/seedu/address/logic/parser/BookShelfParser.java [hunk 5]
```diff
@@ -164,19 +164,19 @@ private Command getCommand(String commandWord, String arguments) throws ParseExc
...
         case ThemeCommand.COMMAND_WORD:
             return new ThemeCommandParser().parse(arguments);
 
         case UndoCommand.COMMAND_WORD:
             return new UndoCommand();
-
+        //@@author 592363789
```
Comment (takuyakanbr) [inline src/main/java/seedu/address/logic/parser/BookShelfParser.java]: Remove the changes in this file...?

File: src/main/java/seedu/address/logic/parser/BookShelfParser.java [hunk 6]
```diff
@@ -164,19 +164,19 @@ private Command getCommand(String commandWord, String arguments) throws ParseExc
...
         case UnlockCommand.COMMAND_WORD:
             return new UnlockCommand(arguments);
-
+        //@@author
```
Comment (takuyakanbr) [inline src/main/java/seedu/address/logic/parser/BookShelfParser.java]: Remove the changes in this file...?

**Comment by takuyakanbr** [comment]
Also, add an extra line at the end of this file: `collated/unused/592363789-unused.md`

**Pull request merged**